### PR TITLE
[SRVKS-937] Updating system deployment override configs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
OCP 4.6+
OSD 4
ROSA

Issue:
https://issues.redhat.com/browse/SRVKS-937

Link to docs preview:
http://file.rdu.redhat.com/abrennan/220622/SRVKS-937/serverless/admin_guide/serverless-configuration.html#knative-serving-CR-system-deployments_serverless-configuration

### CODE REVIEW

File: modules/knative-serving-CR-system-deployments.adoc [hunk 1]
```diff
@@ -6,13 +6,14 @@
 [id="knative-serving-CR-system-deployments_{context}"]
 = Overriding system deployment configurations
 
-You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resource (CR). Currently, overriding default configuration settings is supported for the `replicas`, `labels`, `annotations`, and `nodeSelector` fields.
+You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resource (CR). Currently, overriding default configuration settings is supported for the `resources`, `replicas`, `labels`, `annotations`, and `nodeSelector` fields.
```
Comment (skonto) [inline modules/knative-serving-CR-system-deployments.adoc:9]: The full list of overrides is [here](https://github.com/openshift-knative/serverless-operator/blob/a1ba71728d05ed7b307a2612cd10ac3ede3dedb6/vendor/knative.dev/operator/pkg/apis/operator/v1alpha1/common.go#L220-L251).
Includes also tolerations and affinity, do we reference those somewhere else?

File: modules/knative-serving-CR-system-deployments.adoc [hunk 2]
```diff
@@ -33,6 +34,14 @@ spec:
     replicas: 2
   deployments:
   - name: webhook
+    resources:
+    - container: activator
```
Comment (mgencur) [inline modules/knative-serving-CR-system-deployments.adoc:38]: The deployment is called "webhook" and it only has a container called "webhook". Let's change this line to `- container: webhook`.
The rest looks ok to me.

File: modules/knative-serving-CR-system-deployments.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/knative-serving-CR-system-deployments.adoc:9]: I think that's out of the scope of this PR tbh, the intention was just to add resources.
Feel free to open an additional Jira if there are others missing (please provide examples for each).

**Review by sayjadha — COMMENTED** [review]
LGTM.

**Pull request merged**